test/testcafe/spec/ChallengeOktaVerifyPush_spec.js

```diff
@@ -52,7 +52,9 @@ test
 
     const pageTitle = challengeOktaVerifyPushPageObject.getPageTitle();
     const pushBtn = challengeOktaVerifyPushPageObject.getPushButton();
+    const a11ySpan = challengeOktaVerifyPushPageObject.getA11ySpan();
     await t.expect(pushBtn.textContent).contains('Push notification sent');
+    await t.expect(a11ySpan.textContent).contains('Push notification sent');
     await t.expect(pushBtn.hasClass('link-button-disabled')).ok();
     await t.expect(pageTitle).contains('Get a push notification');
   });
```

Comment:
test below checks for the button to not be disabled and to have a different text. Maybe we should test also that `a11ySpan` is removed? I am not as familiar with how OV works in v2/idx, but looking at what you are doing for v1 it seems we have something similar

Reply:
I thought about adding this test as well, but decided not to because that test is using a different view (more information in [this comment](https://github.com/okta/okta-signin-widget/pull/1691/files/af4fcc6b7407ac7675a97c5bd84d1c2b5f17320f#r582129603)). Since v2 uses multiple views for this scenario, I think we're OK testing only the disabled button state.